python/lsst/ts/atspectrograph/atspec_csc.py

```diff
@@ -127,9 +127,15 @@ async def end_enable(self, data):
             # Check/Report Filter Wheel position.
             state = await self.model.query_fw_status(self.want_connection)
             self.log.debug(f"query_fw_status: {state}")
-            filter_name = str(list(self.model.filters.keys())[int(state[1])])
+
+            filter_name = str(list(self.model.filter_to_enum_mapping_dict.keys())[int(state[1])])
```

Comment:
So, using the type in the variable name is considered an anti-pattern in python. I would recommend renaming `filter_to_enum_mapping_dict` to `filter_to_enum_mapping` or just `filter_to_enum`.